profile/README.md

```diff
@@ -0,0 +1,39 @@
+<p align="center">
+  <img src="https://kong.quero.com/logotype-education-light.svg" alt="Quero Educação"/>
+</p>
+
+> We are technologists and communicators developing solutions so that more people
+have access to quality education.
```

Comment:
esse quality education era pra ser disso mesmo?

> What do we mean by a quality education?
In the statement, ASCD and EI define a quality education: A quality education is one that focuses on the whole child—the social, emotional, mental, physical, and cognitive development of each student regardless of gender, race, ethnicity, socioeconomic status, or geographic location.

Reply:
Eu pensei que poderia ser outro termo também, fui dar uma pesquisada e não achei nada de relevante.
Ai eu cai nesse site ai -> https://theirworld.org/explainers/right-to-education#section-1
Eles usam esse termo também :point_down: ~se tiver errado, não estamos sozinhos~

> Every person is entitled to a quality education without discrimination, which means:
>
> - A compulsory free primary school education for every child
> - Secondary school (including technical training) must be available to everyone - states must work towards providing this for free
> - Higher education must be equally accessible, with countries working towards the goal of making this free
> ...

Talvez mudar para :point_down: ? Não fiz fisk, então vou ficar devendo uma explicação concreta, mas eu achei meio "seco" do jeito que ta :joy: 
```suggestion
have access to a good quality education.
```

Reply:
Roubei do [linkedin](https://www.linkedin.com/company/queroedu) :sweat_smile: 
```
We are technologists and communicators developing solutions to bring together those who
want to learn with experts in teaching.
```

Reply:
nice!
eu tinha feito ~tentado~ uma tradução dinâmica da descrição que ta na nossa org aqui mesmo <img src="https://emoji.slack-edge.com/T024JGC6U/feels-pepo-spin/fb4bf3a86e0f5b4b.gif" width="24px" />

eu gostei bastante desse texto que o @toledompm mandou, vou seguir com ele, tudo bem? qualquer coisa só comentar aqui.

Os commits de fix dessa thread estão aqui:

- ~https://github.com/quero-edu/.github/pull/1/commits/c4cf15109bda491440720373e0380f3b1f212b4e~
- https://github.com/quero-edu/.github/pull/1/commits/4e5953c04b38b59ada2f5a78c32f5e0fc37aa28e